crates/forge_analyzer/src/definitions.rs

```diff
@@ -819,21 +956,119 @@ impl<'cx> FunctionAnalyzer<'cx> {
+            [PropPath::Def(def), PropPath::Static(ref method_name)] => {
+                if let Some((package_name, import_kind)) = self.res.as_foreign_import(def) {
+                    // Star imports that don't have any object call to invoke function
+                    // import * as atlassian_jwt from "atlassian-jwt";
+                    // atlassian_jwt.encode(blah, blah);
+
+                    if import_kind == ImportKind::Star {
+                        let package_found = self.secret_packages.iter().find(|&package_data| {
+                            package_name == package_data.package_name
+                                && *method_name == package_data.identifier
+                                && package_data.method.is_none()
+                        });
+                        if let Some(package) = package_found {
+                            Some(Intrinsic::JWTSign(package.clone()))
+                        } else {
+                            None
+                        }
+                    } else {
+                        // Named or default imports that
+                        // import AES from "crypto-js";
+                        // import {AES} from "crypto-js"
+                        // Aes.encrypt(blah, blah);
+                        let package_found = self.secret_packages.iter().find(|&package_data| {
+                            if let Some(method) = &package_data.method {
+                                return package_name == package_data.package_name
+                                    && import_kind == *package_data.identifier
+                                    && *method_name == *method;
+                            } else {
+                                false
+                            }
+                        });
+                        if let Some(package) = package_found {
+                            Some(Intrinsic::JWTSign(package.clone()))
+                        } else {
+                            None
+                        }
+                    }
+                } else {
+                    None
+                }
+            }
+            // [Def, .., Static] star
+            // [Def, .., Static] named/default
+            // [PropPath::Def(def)]
+            // 1. Named/Default => package matches imported from and ident is the same, (NO METHOD)
+            [PropPath::Def(def)] if self.res.as_foreign_import(def).is_some() => {
+                if let Some((package_name, import_kind)) = self.res.as_foreign_import(def) {
+                    let package = self.secret_packages.iter().find(|&package_data| {
+                        *package_name == *package_data.package_name
+                            && import_kind == *package_data.identifier
+                            && package_data.method.is_none()
+                    });
+                    if package.is_some() {
```

Comment:
```suggestion
                    package.cloned().map(Intrinsic::JWTSign)
```
See [Option::cloned](https://doc.rust-lang.org/std/option/enum.Option.html#method.cloned) and [Option::map](https://doc.rust-lang.org/std/option/enum.Option.html#method.map). Note that tuple structs and enum tuple variants are essentially functions.